Topic5Mysql/src/DataBaseTest.java

```diff
@@ -0,0 +1,74 @@
+import java.net.ConnectException;
+import java.sql.ResultSet;
+
+public class DataBaseTest {
+
+	public static void main(String[] args) throws Exception {
+
+		// Connection
+
+		DataBaseConnection dataBaseConnection = new DataBaseConnection();
+		dataBaseConnection.connect();
+
+		// Test exercise 4
+
+		System.out.println(" ");
+		System.out.println(" ---Excercise 4--- ");
+		System.out.println(" ");
+
+		DataBaseManager dataBaseManager = new DataBaseManager();
+
+		ResultSet resultSet1;
+		ResultSet resultSet2;
+
+		int course_id = 2;
+
+		String query1 = "SELECT course.course_name, teacher.last_name, teacher.first_name " + "FROM course, teacher " + "WHERE course_id ="
+				+ course_id + " AND course.teacher_assigned = teacher_id;";
+
+		String query2 = "SELECT first_name,last_name " + "FROM notes, student " + "WHERE course_id =" + course_id + " AND course_id = "
+				+ course_id + " AND notes.student_id=student.registration_number " + " ORDER BY last_name";

```

Comment:
There are cases when you are able to return the expected result in a complex object (to be printed later) using a single (and complex) query, do you believe that in this case it would be possible?